addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4467,15 +4470,18 @@ export class OdooEditor extends EventTarget {
 
         const selectors = {
             BLOCKQUOTE: this.options._t('Empty quote'),
+            PRE: this.options._t('Code'),
+            DIV: this.options._t('Type "/" for commands'),
+            P: this.options._t('Type "/" for commands'),
             H1: this.options._t('Heading 1'),
             H2: this.options._t('Heading 2'),
             H3: this.options._t('Heading 3'),
             H4: this.options._t('Heading 4'),
             H5: this.options._t('Heading 5'),
             H6: this.options._t('Heading 6'),
-            'UL LI': this.options._t('List'),
-            'OL LI': this.options._t('List'),
-            'CL LI': this.options._t('To-do'),
+            'UL': this.options._t('List'),
+            'OL': this.options._t('List'),
+            'CL': this.options._t('To-do'),
```

Comment:
@lebl-odoo This hint has never worked before in practice, therefore it has never been displayed to the user. To be honest I don't understand why it should say "To-do" instead of "Checklist". I think it looks out of place compared to the other hints*, but I'm fine either way. I just thought this PR might be the right time to ask for your PO opinion on the matter before it is displayed to the user.

* FYI, the `Type "/" for commands` ones are going to be removed from this list due to my other comments in the PR.

Reply:
@lebl-odoo Thanks! @dhba-odoo This new spec is even better because if there is no difference anymore between the types of lists, the selector can simply be `"LI"` and we can remove the special case about it.